pkg/network/tracer/connection/kprobe/manager.go

```diff
@@ -93,6 +93,11 @@ func initManager(mgr *ddebpf.Manager, connCloseEventHandler ddebpf.EventHandler,
 		mgr.Probes = append(mgr.Probes, p)
 	}
 
+	// add Probe for net_dev_queue attached via raw tracepoint
+	mgr.Probes = append(mgr.Probes,
+		&manager.Probe{ProbeIdentificationPair: manager.ProbeIdentificationPair{EBPFFuncName: "raw_tracepoint__net__net_dev_queue", UID: probeUID}, TracepointName: "net_dev_queue", TracepointCategory: "net"},
```

Comment:
It looks like `TracepointCategory` is not necessary for raw tracepoints? https://github.com/DataDog/ebpf-manager/blob/f1cd7d97ecbabc5ead106fcb1c2397125c7fc388/raw_tp.go#L12-L15

Reply:
Should we add parsing in ebpf-manager for raw tracepoints to extract the `TracepointName` similar to how tracepoints are doing?

Reply:
Sounds good. Ill make a PR for that